test/extensions/transport_sockets/tls/utility_test.cc

```diff
@@ -121,7 +124,7 @@ TEST(UtilityTest, GetLastCryptoError) {
 
   ERR_put_error(ERR_LIB_SSL, 0, ERR_R_MALLOC_FAILURE, __FILE__, __LINE__);
   EXPECT_EQ(Utility::getLastCryptoError().value(),
-            "error:10000041:SSL routines:OPENSSL_internal:malloc failure");
+            "error:14000041:SSL routines:SSL routines:malloc failure");
```

Comment:
is this defined somewhere?

Reply:
yeah, it's in openssl's error descriptions (which are different from boringssl ones).